internal/pkg/scaffold/olm-catalog/package_manifest.go

```diff
@@ -127,7 +127,7 @@ func (s *PackageManifest) newPackageManifest() *olmregistry.PackageManifest {
 	return pm
 }
 
-func validatePackageManifest(pm *olmregistry.PackageManifest) error {
+func ValidatePackageManifest(pm *olmregistry.PackageManifest) error {
```

Comment:
Does this function need to be exported?

Reply:
Yes, it will be used to validate package manifests in other packages. I'm going to move it to `internal/util/registry` though. This package location fits better with some additions I'll be making in an OLM-related PR.